tests/conftest.py

```diff
@@ -94,6 +95,11 @@ def deploy(sett_config):
 
     assert want.balanceOf(deployer.address) > 0
 
+    ## Whitelist from CVX Vault
+    veCVX = interface.ISett(strategy.CVX_VAULT())
+    veGov = accounts.at(veCVX.governance(), force=True)
+    veCVX.approveContractAccess(strategy, {"from": veGov})
```

Comment:
Best to only do this for the cvxCrv strat
```python
if sett_config.strategyName == "StrategyCvxCrvHelper":
    ...
```

Reply:
<img width="570" alt="Screenshot 2022-01-05 at 17 13 09" src="https://user-images.githubusercontent.com/13383782/148250570-4af14ce4-78c1-4f18-8c91-4d487e927375.png">
Great catch was just adding it